### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sayaliM0412**
feat: `CONTRIBUTING.md`: update readme to Add an Extension

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,3 +15,24 @@
...
+```
+3. Edit `internal/app/packages.json` with the extension details.
+```
+{
+    "name": "extension-name",
+    "category": "extension", // or "driver"
+    "versions": []
+  }
+```
+4. From the Terminal run command : `make generateExtensionPackages`. It will automatically update the `internal/app/packages.json` file with values for `algorithm`, `checksum`, `liquibaseCore`, `path`
```
Comment (jnewton03) [inline CONTRIBUTING.md:38]: instead of running this from the terminal could we have it trigger the "update packages" workflow on PR?

Comment (sayaliM0412) [reply CONTRIBUTING.md:38]: we can run the workflow from a branch from here https://github.com/liquibase/liquibase-package-manager/actions/workflows/nightly-update-packages.yml to make it happen. Correct me if I am wrong @mcred

Comment (mcred) [reply CONTRIBUTING.md:38]: Adding the following or something similar to `nightly-update-packages` should cover it.
```
on:
  workflow_dispatch:
  schedule:
    - cron: '5 6 * * *'
  push:
    branches:
      - master
    paths:
      - "cmd/populator/**"
```

Comment (sayaliM0412) [reply CONTRIBUTING.md]: Thanks @mcred 
working as expected.
Example : https://github.com/liquibase/liquibase-package-manager/actions/runs/9976289363/job/27568093384
Automatic PR : https://github.com/liquibase/liquibase-package-manager/pull/404/files

**Comment by jnewton03** [comment]
@mcred any chance you can review this please?

File: .github/workflows/nightly-update-packages.yml [hunk 2]
```diff
@@ -2,6 +2,12 @@ name: Update Packages
 
 on:
   workflow_dispatch:
+    pull_request:
```
Comment (mcred) [inline .github/workflows/nightly-update-packages.yml:5]: This formatting might make GHA unhappy. Pull request should be on the same tab break as workflow_dispatch and schedule. should `- reopened` also be in the list?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,3 +15,24 @@
...
+    category:    Extension,
+    url:         "https://repo1.maven.org/maven2/org/liquibase/ext/liquibase-commercial-dynamodb",
+    artifactory: Maven{},
+}
+```
+2. Edit `internal/app/packages.json` with the extension details.
+```
+{
+    "name": "extension-name",
+    "category": "extension", // or "driver"
```
Comment (mcred) [inline CONTRIBUTING.md:34]: this can also be `pro` and the categories should match with the populator (which is an enum not a string)

**Comment by sayaliM0412** [comment]
@jnewton03 @mcred :are we good to merge this PR?

**Review by mcred — APPROVED** [review]

**Pull request merged**